xtext-editor/src/main/src-gen/gov/ca/water/wresl/xtext/editor/wreslEditor/impl/AddImpl.java

```diff
@@ -28,8 +22,8 @@
  * <p>
  * The following features are implemented:
  * <ul>
- *   <li>{@link gov.ca.dwr.wresl.xtext.editor.wreslEditor.impl.AddImpl#getM1 <em>M1</em>}</li>
- *   <li>{@link gov.ca.dwr.wresl.xtext.editor.wreslEditor.impl.AddImpl#getM2 <em>M2</em>}</li>
+ *   <li>{@link AddImpl#getM1 <em>M1</em>}</li>
```

Comment:
Again this might need to be the full package + class given this is documentation